sensu/plugins/check-l3-router.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+HOSTNAME=`hostname`
+for IFACE in $(ifquery --list); do
```

Comment:
How does this logic handle the difference between public ucarp and private ucarp addresses?  In this case if you have at least one, you will set UCARP="owned"